### PR TITLE
[JENKINS-48831] New/fixed version of js-lib core-asset modules

### PR DESCRIPTION
**Pull request by tfennelly**
See [JENKINS-48831](https://issues.jenkins-ci.org/browse/JENKINS-48831).

Depends on https://github.com/jenkinsci/js-libs/pull/4

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by jglick** [comment]
Not a werk-in-progress merely because it might need snapshot dependency updates; that is something that can be done by a merger.

File: war/pom.xml [hunk 1]
```diff
@@ -41,6 +41,7 @@ THE SOFTWARE.
     <JENKINS_HOME>${basedir}/work</JENKINS_HOME>
     <contextPath>/jenkins</contextPath><!-- context path during test -->
     <port>8080</port><!-- HTTP listener port -->
+    <js.libs.version>2.0.0-SNAPSHOT</js.libs.version>
```
Comment (jglick) [inline war/pom.xml]: Suggest `js-libs.version` to match repo name.

**Comment by daniel-beck** [comment]
On-holding until upstream release.

**Comment by daniel-beck** [comment]
@tfennelly Do you plan to finish this? I think there's no release of the dependency yet?

File: war/pom.xml [hunk 2]
```diff
@@ -41,6 +41,7 @@ THE SOFTWARE.
     <JENKINS_HOME>${basedir}/work</JENKINS_HOME>
     <contextPath>/jenkins</contextPath><!-- context path during test -->
     <port>8080</port><!-- HTTP listener port -->
+    <js-libs.version>2.0.0-SNAPSHOT</js-libs.version>
```
Comment (oleg-nenashev) [inline war/pom.xml]: It has not been released so far

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**